modules/dr-restoring-cluster-state.adoc

```diff
@@ -90,7 +90,37 @@ $ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard"
 +
 The output of this command should be empty. If it is not empty, wait a few minutes and check again.
 
-.. Move the etcd data directory to a different location:
+.. Move the existing Kubernetes Controller Manager pod file out of the kubelet manifest directory by using the following command:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/kube-controller-manager-pod.yaml /tmp
+----
+
+.. Verify that the Kubernetes Controller Manager containers are stopped by running the following command:
++
+[source,terminal]
+----
+$ sudo crictl ps | grep kube-controller-manager | egrep -v "operator|guard"
+----
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+.. Move the existing Kubernetes Scheduler pod file out of the kubelet manifest directory by using the following command:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/kube-scheduler-pod.yaml /tmp
+----
+
+.. Verify that the Kubernetes Scheduler containers are stopped by using the following command:
++
+[source,terminal]
+----
+$ sudo crictl ps | grep kube-scheduler | egrep -v "operator|guard"
+---- 
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+.. Move the etcd data directory to a different location with the following command:
```

Comment:
"Example" should be mentioned as the command is just that based on the `tmp` target location. The preceeding text does not suggest a location limitation.